SFM.py

```diff
@@ -184,15 +185,15 @@ def adj_list(self, matches=None):
                     self.adj[i][j] = 1
 
         return self.adj, pairs
-
     
-    def reconstruction(self):
-        if self.K is None:
-            raise AssertionError("Error: Camera matrix K not defined")
-        
-        # initial rec
-        # best_pair = sfm_helpers.Rec.best_pair(self.adj, self.good, self.kp, self.K)
-        
+    def get_camera_info(self):
+        files = os.listdir(self.src)
+        for file in files:
```

Comment:
i think u can use self.itoimg to get the image file names instead of looking at the path again